### ISSUE TITLE
Add some settings

### ISSUE THREAD

**Issue by nymous** [WikiJourney/wikijourney_app#21]
Still no idea what to put there, but still...

**Comment by nymous** [comment]
We should put default range/# of POI, some way to filter between types of POIs...

**Comment by sylvainar** [comment]
:+1:

**Comment by sylvainar** [comment]
What about the language for API requests ? It's not obliged to always be the same that the one used for the app.

**Comment by gjedeer** [comment]
Language! For cities in my country I want to make requests in my language - local Wikipedia has much more resources. For travelling abroad, usually English has more POIs

**Comment by sylvainar** [comment]
Yup, you're right. Actually, this functionality is actually implemented on the API and on the website ( https://www.wikijourney.eu/ ), so in the future it's gonna be on the app too.
